.devcontainer/compose.yml

```diff
@@ -31,7 +31,7 @@ services:
     entrypoint: mkdocs
     command: serve --dev-addr "0.0.0.0:8000"
```

Comment:
Better to change this to 8001 for consistency?

Reply:
This is the internal port... but sure? I don't know if `mkdocs` expects to be running on a certain port or not.

Reply:
Changed in [ff47855](https://github.com/cal-itp/benefits/pull/598/commits/ff47855768355d154513d267f01bbe34116c253e), went back to a dynamic local port but `8001` for the internal port to be clear we're different from the app.